pkg/controller/kogitoinfra/status/manage_status.go

```diff
@@ -63,7 +69,7 @@ func ManageDependenciesStatus(instance *v1alpha1.KogitoInfra, cli *client.Client
 	updatedInfinispan := ensureInfinispan(instance, cli)
 	if updatedInfinispan {
 		log.Infof("Updating status of Kogito Infra instance %s ", instance.Status)
-		if updatedErr := kubernetes.ResourceC(cli).UpdateStatus(instance); updatedErr != nil {
+		if updatedErr := kubernetes.ResourceC(cli).Update(instance); updatedErr != nil {
```

Comment:
why not keep status? is this Update method used for anything else?

Reply:
`UpdateStatus` only works if `subresource` is enabled. `subresource` is not compatible with Kubernetes 1.11, hence OpenShift 3.11. That's why we are using `Update` instead.

Reply:
See:
https://github.com/kiegroup/kogito-cloud-operator/pull/112/files#diff-4376e4cda4738fcf894e5e795c63f398L13
and:
https://issues.jboss.org/browse/KOGITO-634